test/fw/ptl/utils/pbs_testsuite.py

```diff
@@ -155,6 +156,9 @@ def timeout(val):
     """
     Decorator to set timeout value of test case
     """
+    if val < DEFAULT_TESTCASE_TIMEOUT:
+        val = DEFAULT_TESTCASE_TIMEOUT
```

Comment:
why are you doing this? If a test case explicitly says it needs x seconds of timeout, why do you wanna change it to be a minimum of DEFAULT_TESTCASE_TIMEOUT ? I think we should leave custom timeouts alone.

Reply:
@agrawalravi90 , previously timeout value was 180 and there are some tests have value less than 600 and greater than 180 in timeout tag, to make affect on those tests also i added above condition.